app/controllers/key_pairs_controller.rb

```diff
@@ -60,11 +89,13 @@ def update
 
     respond_to do |format|
       if @key_pair.update_attributes(key_pair_params)
-        format.html { redirect_to @key_pair, notice: 'Key pair was successfully updated.' }
+        format.html { redirect_to @key_pair,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.